src/test/e2e/Tests/e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { Browser, Page } from 'puppeteer';
+
+const puppeteer = require('puppeteer');
+const HomePage = require('../PageObjects/HomePage');
+const homePage = new HomePage;
+let browser: Browser;
+let page: Page;
```

Comment:
if we return our own pages we can use those and the standard browser object in ours instead of the browsers new pages